### ISSUE TITLE
GreenKeeper not properly integrated?

### ISSUE THREAD

**Issue by fadeenk** [AlKass/PiTrol#18]
GreenKeeper's branch dosent seem to be merged in and there are 2 packages that are already outdated but greenkeeper hasnt created PRs for them.

Package        Current  Wanted  Latest  Location
react-router     2.8.1   2.8.1   3.0.2  mypicontrol
react-scripts    0.2.1   0.2.1   0.9.0  mypicontrol

**Comment by AlKass** [comment]
what level of access do you need to investigate the problem? I'm currently logged in to guthub from a non-official mobile app

**Comment by fadeenk** [comment]
Whichever level that gives you access to the repo's settings

**Comment by AlKass** [comment]
I'll add you as a collaborator and if that still doesn't work i can transfer ownership to you and then you can return it back to me

**Comment by fadeenk** [comment]
Ok looks like it is still missing :( i still dont have the settings tab

**Comment by AlKass** [comment]
It won't let me transfer the repo's ownership to you because you already have a repo with an identical name (the fork you made). either rename your fork or get rid of it so I can move this to you

**Comment by fadeenk** [comment]
I removed my fork

**Comment by AlKass** [comment]
you are now the owner of the repo. Please make all the required adjustments and move it back to me when you're done. Thanks

Fadi

**Comment by fadeenk** [comment]
Got coveralls to work however couldnt figure out what is going on with greenkeeper

**Comment by AlhasanIQ** [comment]
is this issue still relevant ?

**Comment by fadeenk** [comment]
Yes, it seems to be connected but it is not working properly so i am not sure what is wrong. Thats why I added the help wanted flag

**Comment by AlhasanIQ** [comment]
i dp mpt suppose any of the contributors have any idea about greenkeeper / coveralls 😃

**Comment by fadeenk** [comment]
Coveralls is setup and functional however after the transfer things might break but not sure on that.

**Comment by AlKass** [comment]
@AlhasanIQ 
I do. I don't know how to set them up though

**Comment by AlKass** [comment]
@fadeenk 
Things have broken. Checked earlier
also if you happen to push a fix, make sure you update the links in the badges in README.md

**Comment by fadeenk** [comment]
I see, I will need an admin access then to fix things... I can fix the badges easily but the integration will require me to have access to the settings for the repo

**Comment by AlKass** [comment]
@fadeenk 
owner role granted. try now

**Comment by fadeenk** [comment]
Fixed travis, enabled green keeper (still not sure it is working)
coveralls is not syncing the repos so could not enable it
